integrations/extensions/starter-kits/language-model-personalization/README.md

```diff
@@ -0,0 +1,95 @@
+
+1. The `Invoke GPT Chat Completion API` action is recognized. The assistant responds with a personalized response.
+
+Here is an example of what the output can look like:
+
+<img src="./assets/sample.png" width="300"/>
+
+The behavior of the model can vary, so the output you get for this prompt may not match the one seen here.
+
+### Details about the Close bank account sample action
+
+| Step | Description |
+| --- | --- |
+| 1 | Sets hardcoded values for three variables: `customer_name`, `account_type`, and `email`. You can modify these values as you want. In a real application, you would probably get them from a database if your user was logged in to the site where your assistant is deployed.  |
+| 2 | Uses the variable `model_prompt` to ask OpenAI to pretend to be a bank and personalize the response based on the customer's reason from step 1. Set to use the `Invoke GPT Chat Completion API` action, which uses the `gpt-3.5-turbo` model. Depending on your OpenAI plan, you can modify to use the `Invoke GPT Completion API` action, which uses the more expensive `text-davinci-003` model. |
+| 3 | Uses the variable `model_response` to show the response from OpenAI to the customer. |
+| 4 | Example options step |
+| 5 | Example connect to agent step |
+
+The `Close bank account` action currently connects to OpenAI using the `Invoke GPT Chat Completion API` action, but you can change the action call out in step 2 of this action to call a different model-invoking action instead.  The model-invoking action needs to take in the `model_prompt` session variable, prompt the model using that prompt, and then set `model_responded` to true or false depending on whether the model provided a response and `model_response` to that response if it did.  The `Invoke GPT Completion API` action meets this requirement too, so you change the action call out in step 2 of `Close bank account` to call `Invoke GPT Completion API` action if you want to try calling the slightly higher-quality (but much more expensive) `text-davinci-003` model.  Furthermore, if you create an action that invokes some _other_ language model using `model_prompt` as input and `model_responded` and `model_response` as outputs, you can change the action call out to call that _other_ model-invoking action instead.
+
+## Responsible Use
+
+The full prompt expression in step 2 is:
+
+> "Pretend you are a bank and that the customer wants to close their bank account because " + ${step_457} + ". address the customer directly in the chat window and convince them to stay with the bank? Personalize with any info they provide. If you don't know the customer's name then address the customer generically. Here are some details about the customer, if any of the values are null just ignore and don't mention them: customer name: ${customer_name}, and customer email ${email}, and account type: ${account_type}. Limit your response to 400 characters without trailing off, don't mention this to the customer."
+
```

Comment:
Not a big deal, but this part of the prompt would be grammatically conformant if it starts with a capitalized word and ends with a period instead of a question mark. 
```
address the customer directly in the chat window and convince them to stay with the bank?
```